SimPEG/Regularization.py

```diff
@@ -470,58 +459,6 @@ def _nC_residual(self):
         else:
             return self.nP
 
-    @property
-    def mesh(self):
```

Comment:
yep. But I don't think there is any need to access it from the regularization itself, that is just a potential point on confusion